terminus-ui/autocomplete/src/autocomplete.component.spec.ts

```diff
@@ -350,14 +350,13 @@ describe(`TsAutocompleteComponent`, function() {
 
         // Verify the selection DID work
         chips = getAllChipInstances(fixture);
-        expect(chips.length).toEqual(1);
+        expect(chips.length).toEqual(2);
```

Comment:
has the seeded value got updated? (used to be 1 element in an array)

Reply:
it shouldn't have been 1. the method is testing to allow duplicates. so 1 already exists (florida) and then we pick the same one again. so, 2 chips should be present. i guess before it wasn't updating the chips properly.